ci/azure-pipelines.yml

```diff
@@ -27,22 +27,38 @@ jobs:
         CODECOV_UPLOAD_TOKEN: $(CODECOV_UPLOAD_TOKEN)
       displayName: Upload coverage to Codecov
 
-  - job: BDDTest
+  - job: BDDTest_lscc
     condition: in(variables['test-type'],'','bdd-test') # Run if test-type is not set or if test-type is 'bdd-test'
     pool:
       vmImage: ubuntu-18.04
     strategy:
-      parallel: 6
+      parallel: 3
     timeoutInMinutes: 60
     steps:
     - template: azp-dependencies.yml
     - checkout: self
     - script: make bddtests
```

Comment:
you might want to explicitly put the fixtures version for the lscc run.

Reply:
Sounds good - in my next PR.